src/ethereum_test_tools/vm/opcode.py

```diff
@@ -198,178 +192,4947 @@ class Opcodes(Opcode, Enum):
+    """
 
     POP = Opcode(0x50, popped_stack_items=1)
+    """
+    POP(y)
+    ----
+
+    Description
+    ----
+    Remove item from stack
+
+    Inputs
+    ----
+    - y: a stack item
+
+    Outputs
+    ----
+    - None
+
+    Inputs
+    ----
+    - None
+
+    Fork
+    ----
+    Frontier
+
+    Gas
+    ----
+    2
+
+    Source: [evm.codes/#50](https://www.evm.codes/#50)
+    """
+
     MLOAD = Opcode(0x51, popped_stack_items=1, pushed_stack_items=1)
+    """
+    MLOAD(offset) = value
+    ----
+
+    Description
+    ----
+    Load word from memory
+
+    Inputs
+    ----
+    - offset: offset in the memory in bytes
+
+    Outputs
+    ----
+    - value: the 32 bytes in memory starting at that offset. If it goes beyond its current size
+      (see MSIZE), writes 0s
+
+    Fork
+    ----
+    Frontier
+
+    Gas
+    ----
+    3 + memory_expansion_cost
+
+    Source: [evm.codes/#51](https://www.evm.codes/#51)
+    """
+
     MSTORE = Opcode(0x52, popped_stack_items=2)
-    MSTORE8 = Opcode(0x53, popped_stack_items=2)
-    SLOAD = Opcode(0x54, popped_stack_items=1, pushed_stack_items=1)
-    SSTORE = Opcode(0x55, popped_stack_items=2)
-    JUMP = Opcode(0x56, popped_stack_items=1)
-    JUMPI = Opcode(0x57, popped_stack_items=2)
+    """
+    MSTORE(offset, value)
+    ----
+
+    Description
+    ----
+    Save word to memory
+
+    Inputs
+    ----
+    - offset: offset in the memory in bytes.
+    - value: 32-byte value to write in the memory.
+
+    Outputs
+    ----
+    - None
+
+    Fork
+    ----
+    Frontier
+
+    Gas
+    ----
+    3 + memory_expansion_cost
+
+    Source: [evm.codes/#52](https://www.evm.codes/#52)
+    """
+
+    MSTORE8 = Opcode(0x53, popped_stack_items=2)
+    """
+    MSTORE8(offset, value)
+    ----
+
+    Description
+    ----
+    Save byte to memory
+
+    Inputs
+    ----
+    - offset: offset in the memory in bytes
+    - value: 1-byte value to write in the memory (the least significant byte of the 32-byte stack
+      value
+
+    Fork
+    ----
+    Frontier
+
+    Gas
+    ----
+    3 + memory_expansion_cost
+
+    Source: [evm.codes/#53](https://www.evm.codes/#53)
+    """
+
+    SLOAD = Opcode(0x54, popped_stack_items=1, pushed_stack_items=1)
+    """
+    SLOAD(key) = value
+    ----
+
+    Description
+    ----
+    Load word from storage
+
+    Inputs
+    ----
+    - key: 32-byte key in storage
+
+    Outputs
+    ----
+    - value: 32-byte value corresponding to that key. 0 if that key was never written before
+
+    Fork
+    ----
+    Frontier
+
+    Gas
+    ----
+    100
+
+    Source: [evm.codes/#54](https://www.evm.codes/#54)
+    """
+
+    SSTORE = Opcode(0x55, popped_stack_items=2)
+    """
+    SSTORE(key, value)
+    ----
+
+    Description
+    ----
+    Save word to storage
+
+    Inputs
+    ----
+    - key: 32-byte key in storage
+    - value: 32-byte value to store
+
+    Outputs
+    ----
+    - None
+
+    Fork
+    ----
+    Frontier
+
+    Gas
+    ----
+    100
```

Comment:
```suggestion
    set: 20000
    reset: 2900
```
There's a lot of nuances to the actual gas cost of this opcode, but we can leave this snippet as this for now.